build.gradle

```diff
@@ -167,22 +167,27 @@ eclipseClasspath {
     }
 }
 
```

Comment:
Yup [looks like it works](https://travis-ci.org/TEAMMATES/teammates/builds/235932910) :)